lib/osv/sources.py

```diff
@@ -173,3 +188,11 @@ def push_source_changes(repo, commit_message, git_callbacks):
 def git_author():
   """Get the git author for commits."""
   return pygit2.Signature('OSV', AUTHOR_EMAIL)
+
+
+def sha256(yaml_path):
+  """Computes sha256 sum."""
+  hasher = hashlib.sha256()
+  with open(yaml_path, 'rb') as f:
```

Comment:
should it check if file exists, and return none, logerror if not ?

Reply:
This will be up to the caller. We should be getting an exception if this is getting called with a non-existent path.